marketplace_builder/views/pages/tutorials/form-configurations/configuration-options.liquid

```diff
@@ -0,0 +1,193 @@
+
+{% include 'alert/warning', content: 'MarketplaceKit currently do not raise an error when two form configuration have the same name - last synced form will be displayed' %}
+
+### resoruce
+`resource` is either the name of your CustomModelType or one of predefined PlatformOS clasess. For the full list of classes visit [REST API reference page](api-reference/rest-api).
+
+Examples:
+* [Minimal form example](/tutorials/form-configurations/introduction#minimal-form-example) to see the example of **dynamic resource** based on CustomModelType
+
+* [Creating a Sign Up Form](/tutorials/users/creating-sign-up-form#step-2-configure-forms) for the **predefined class resource** example.
+
+
+## List of optional form configuration options
+
+### api_call_notifications
+`api_call_notifications` is array of api call notifications names that will be executed after the form is successfully submitted.
+
+Example from [Creating an API Call Notification](/tutorials/notifications/creating-api-call-notification
+):
+```yml
+api_call_notifications:
+  - ping_example_com_on_user_sign_up
+```
+
+### async_callback_actions
+`async_callback_actions` accepts two parameters:
+* delay - Integer, number of minutes added to the moment of form submition before the code defined in `content` is executed.
+* content - String, Liquid Template code executed asynchronusly. There is no timeout set for processing this code so it is the right place for more time-consuming calculations.
+
+Example:
+{% raw %}
+```yml
+async_callback_actions:
+  delay: 1
+  content: >
+    {% log "Hello from background" %}
+```
+{% endraw %}
+
+The example above prints "Hellow from background" string to the output of [`marketplace-kit logs staging`](/tutorials/qa/creating-marketplace-kit-logs).
+
+### authorization_policies
+`authorization_policies` Array, list of names of [Authorization Policies](/tutorials/authorization-policy/authorization-policy) that are protecting the form from unauthorized submittion.
+
+### callback_actions
+`callback_actions` String, Liquid Template code that is executed as soon as form is succesfully validated.
+
+### configuration
+`configuration` Object, whitelist of fields that are allowed to pass to the form object together with the validation rules for each of defined fields. Every property added to the configuration is accessible in `form_builder` variable and has the following configuration options:
+** validation - the learn more go to [validation tutorial](/tutorials/form-configurations/validation)
+
+
+**property_options**
+  * virtual - Virtual fields come in handy when there’s no direct mapping to a model attribute or when you plan on displaying but not processing a value.
+
+    Example:
```

Comment:
Weird indentation, might break something as markdown is whitespace sensitive

Reply:
locally if not indented this way it was breaking the list *

Reply:
What is listed in here?